.github/workflows/docker-build-test.yml

```diff
@@ -3,6 +3,9 @@ name: docker-build-test
 on:
   workflow_dispatch:
   pull_request:
+  merge_group:
+    types:
+      - checks_requested
```

Comment:
Should we also add the `release` check to the `merge_group` checks?

Reply:
Right now, `release` only runs on `push`, not `pull_request`. I think we *could* run it on `merge_group`, though potentially this will try to release the same version of the code twice on the test PyPI - which might cause spurious failures? I would rather not change more configs now, get the merge queue stuff working with a close mirror of our existing `pull_request:` flows, and then add `release` to the flow as a separate experiment.

Reply:
Ah okay, sounds good.